napari/layers/base/base.py

```diff
@@ -252,33 +253,42 @@ def __init__(
         self._thumbnail = np.zeros(self._thumbnail_shape, dtype=np.uint8)
         self._update_properties = True
         self._name = ''
-        self.events = EmitterGroup(
-            source=self,
-            auto_connect=False,
-            refresh=Event,
-            set_data=Event,
-            blending=Event,
-            opacity=Event,
-            visible=Event,
-            select=Event,
-            deselect=Event,
-            scale=Event,
-            translate=Event,
-            rotate=Event,
-            shear=Event,
-            affine=Event,
-            data=Event,
-            name=Event,
-            thumbnail=Event,
-            status=Event,
-            help=Event,
-            interactive=Event,
-            cursor=Event,
-            cursor_size=Event,
-            editable=Event,
-            loaded=Event,
-            _ndisplay=Event,
+
+        _events = dict.fromkeys(
```

Comment:
necessary if we want something (like `LayerGroup`) to be able to be a Layer without clobbering superclass events